apps/server/src/modules/tldraw/helper/test-helper.ts

```diff
@@ -0,0 +1,22 @@
+import WebSocket from 'ws';
+
+export class TestHelper {
```

Comment:
Can you please move the file to tldraw/testing/test-connection.ts 
Rename class similar to TestConnection.